test/runner.py

```diff
@@ -477,7 +477,7 @@ def parse_args():
                       help='Command to launch web browser in which to run browser tests.')
   parser.add_argument('--headless', action='store_true',
                       help='Run browser tests in headless mode.', default=None)
-  parser.add_argument('--browser-auto-config', type=bool, default=True,
+  parser.add_argument('--browser-auto-config', action=argparse.BooleanOptionalAction, default=None,
```

Comment:
Hmm.. i've never seen that before.   Does it magically provide the `--no-` variant?

Reply:
Yeah, just learned about it the other day.

Reply:
Err...I guess it's too new. Not available until 3.9.